### PR TITLE
removed links from website/layouts/partials/footer.html

### PR DESCRIPTION
**Pull request by AlexVCS**
working on #1839, going to check in GitHub discussion how exactly to run the website locally (haven't been able to successfully yet)

### CODE REVIEW

**Comment by AlexVCS** [comment]
Screenshot 2023-09-26 at 12 48 33 PM

Got the project running locally and can confirm that this change retains the icons on the right only.

**Comment by AkihiroSuda** [comment]
Thanks, but please sign the commit for DCO
https://github.com/apps/dco

(run `git commit -a -s --amend`, and make sure that the `Signed-off-by: NAME <EMAIL>` line with your real name is included in the commit message)

File: website/hugo.toml [hunk 1]
```diff
@@ -167,16 +167,6 @@ enable = false
...
-[[params.links.user]]
-  name = "Discussion"
-  url = "https://github.com/lima-vm/lima/discussions"
-  icon = "fab fa-github"
-  desc = "Discussion and help from your fellow users"
-[[params.links.user]]
-  name = "Slack"
-  url = "https://slack.cncf.io/"
-  icon = "fab fa-slack"
-  desc = "Chat with other project users"
```
Comment (AkihiroSuda) [inline website/hugo.toml:170]: https://deploy-preview-1842--lima-vm.netlify.app/community/

> ## Learn and Connect
> Using or want to use Lima? Find out more here:

↑ This sections is now empty

Comment (AlexVCS) [reply website/hugo.toml:170]: yes it is empty now. I thought the idea was to remove multiple instances of the icons?

**Comment by AlexVCS** [comment]
Sorry for the force push but I think this is a better way to just remove the icons from the footer.

**Comment by AkihiroSuda** [comment]
Looks good, but please squash commits

**Comment by AlexVCS** [comment]
did that squash work correctly?

**Comment by AkihiroSuda** [comment]
> did that squash work correctly?

The squash worked, but the commit message seems broken.
`Signed-off-by...` has to be a separate line. (See other PRs)

**Comment by AlexVCS** [comment]
> > did that squash work correctly?
> 
> The squash worked, but the commit message seems broken. `Signed-off-by...` has to be a separate line. (See other PRs)

ok thanks, I think it should be fine now

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Review by balajiv113 — APPROVED** [review]

**Comment by balajiv113** [comment]
Some reason preview not available

**Comment by AlexVCS** [comment]
> Some reason preview not available

I’ve been learning a lot with this simple change. Does the preview need to be available?

**Pull request merged**